uportal-war/src/main/java/org/jasig/portal/security/provider/DefaultPermissionPolicy.java

```diff
@@ -1,139 +0,0 @@
-/**
- * Licensed to Apereo under one or more contributor license
- * agreements. See the NOTICE file distributed with this work
- * for additional information regarding copyright ownership.
- * Apereo licenses this file to you under the Apache License,
- * Version 2.0 (the "License"); you may not use this file
- * except in compliance with the License.  You may obtain a
- * copy of the License at the following location:
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-package org.jasig.portal.security.provider;
-
-import java.util.Date;
-import java.util.Iterator;
-
-import org.jasig.portal.AuthorizationException;
-import org.jasig.portal.groups.GroupsException;
-import org.jasig.portal.groups.IGroupMember;
-import org.jasig.portal.permission.IPermissionActivity;
-import org.jasig.portal.permission.IPermissionOwner;
-import org.jasig.portal.permission.target.IPermissionTarget;
-import org.jasig.portal.security.IAuthorizationPrincipal;
-import org.jasig.portal.security.IAuthorizationService;
-import org.jasig.portal.security.IPermission;
-import org.jasig.portal.security.IPermissionPolicy;
-
-/**
- * Implements a strategy for answering the basic authorization question: does the
- * <code>principal</code> have permission to perform the <code>activity</code> on
- * the <code>target</code>.
- *
- * @author Dan Ellentuck (de3@columbia.edu)
- * @version $Revision$
- */
-public class DefaultPermissionPolicy implements IPermissionPolicy {
```

Comment:
Why are we getting rid of this file?  (I can imagine some good reasons... but I don't see a reason in the ticket or the PR.)